x-pack/lib/filters/geoip/database_manager.rb

```diff
@@ -26,12 +26,14 @@
 #   while `offline` is for static database path provided by users
 
 module LogStash module Filters module Geoip class DatabaseManager
```

Comment:
In a sequence of modules in one line probably is best to separate class and module chain, to have structure like:
```ruby
module LogStash module Filters module Geoip
  class DatabaseManager
  ...
  end
end; end; end
```

or go with: 
```ruby
class LogStash::Filters::Geoip::DatabaseManager
...
end
```